Dangerfile

```diff
@@ -53,4 +53,4 @@ labels_checker.check(
 )
 
 # runs the milestone check if this is not a WIP feature and the PR is against the main branch or the release branch
-milestone_checker.check_milestone_due_date(days_before_due: 2) if (github_utils.main_branch? || github_utils.release_branch?) && !github_utils.wip_feature?
+milestone_checker.check_milestone_due_date(days_before_due: 2, report_type_if_no_milestone: :error) if (github_utils.main_branch? || github_utils.release_branch?) && !github_utils.wip_feature? && !github.pr_labels.include?('milestone-exemption')
```

Comment:
<table data-meta="generated_by_pr-check">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/woocommerce/woocommerce-ios/blob/3d362428ffee5db53f5b3f00f1a1bcca61179d65/Dangerfile#L56"></span>Layout/LineLength: Line is too long. [245/180]</td>
    </tr>
  </tbody>
</table>